docs/requester_pays.rst

```diff
@@ -0,0 +1,195 @@
+that Principal the "Role" of **Service Usage Consumer**, which enables it to make
+billed requests on the behalf of the currently selected project (which should be the
+one you created at the beginning of this process -- the project should be visible in the
+dropdown menu near the upper-left corner of the page).
+
+Click on the little pencil icon in the right column of the row associated with your
+Google ID to "edit principal".
+
+.. image:: images/04-iam-admin.png
+
+This should give you the option to associate a role with a particular combination of
+Principal and Project.
+
+.. image:: images/05-add-a-role.png
+
+Click on the "Role" field and type in "Service Usage Consumer"
+and it should filter the very long list of available roles down to a few, one of which
+is "Service Usage Consumer". Select that role, and click on the Save button.
+
+.. image:: images/06-add-service-usage-consumer.png
+
+
+Check your GCP credit
+~~~~~~~~~~~~~~~~~~~~~
+Google provides new accounts with $300 in free credits so you can experiment a bit
+without spending any of your own money. Check out the
+`docs on the free trial period <https://cloud.google.com/free/docs/gcp-free-tier>`__
+to understand how it works.
+
+To check in on your remaining credits or overall billing status you can go to the
+Billing Account Overview and click on the "Credit details" link in the Credits box on
+the right hand side of the page:
+
+.. image:: images/07-billing-account-overview.png
+
+This will show you a table of any existing GCP credits that have been granted to your
+account:
+
+.. image:: images/08-credits-status.png
+
+Set up the Google Cloud SDK
+---------------------------
+
+Now that you have a GCP project with billing enabled, and your Google ID is empowered to
+make use of it, you need to install software that allows your computer to communicate
+with GCP and make authenticated requests on your behalf. This is part of the Google
+Cloud software Development Kit (SDK).
+
+Install the `gcloud utilities <https://cloud.google.com/sdk/docs/install>`__ on your
+computer. There are several ways to do this. We recommend using ``conda`` or its faster
+sibling ``mamba``, since you'll likely be using this catalog in the context of an
+existing conda environment. If you're not using `conda` environments, there are other
+ways to install the Google Cloud SDK explained in the link above.
+
+.. code:: bash
+
+  conda install -c conda-forge google-cloud-sdk
+
+Log into the account you used to create your new project above by running:
+
+.. code:: bash
+
+  gcloud auth login
+
+Initialize the ``gcloud`` command line interface and select the project you just created
+as the default project. This allows the project to be used for requester pays access
+through the command line or in software (like an Intake catalog) that has access to
+shell environment variables.
+
+If it asks you whether you want to "re-initialize this configuration with new settings"
```

Comment:
I was given this prompt after running `gcloud init`. Confusing here because `gcloud init` is hanging out after this paragraph. I was unsure if I should run `gcloud init` right after `gcloud auth login`.